addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_image.py

```diff
@@ -122,9 +118,7 @@ def __gather_name(export_image, export_settings):
     if len(filepaths) == 1:
         filename = os.path.basename(list(filepaths)[0])
         name, extension = os.path.splitext(filename)
-        if extension.lower() in ['.png', '.jpg', '.jpeg']:
-            if name:
-                return name
+        return name
```

Comment:
Attempting to remove as much as possible that is assuming a particular filetype. This check seems superfluous anyway.